File: android/src/main/kotlin/com/camerakit/camerakit_flutter/InvokeMethods.kt
//
//  InvokeMethods.kt
//  com.camerakit.camerakit_flutter
//
//  Copyright © 2023 DevCrew I/O
//

package com.camerakit.camerakit_flutter

interface InvokeMethods {}
object InputMethods: InvokeMethods {
    const val GET_GROUP_LENSES = "getGroupLenses"
    const val OPEN_CAMERA_KIT = "openSnapCameraKit"
    const val OPEN_SINGLE_LENS = "openSingleLens"

}

object OutputMethods: InvokeMethods {
    const val RECEIVED_LENSES = "receivedLenses"
    const val CAMERA_KIT_RESULTS = "cameraKitResults"

}

File: lib/invoke_methods.dart
//
//  invoke_methods.dart
//  com.camerakit.camerakit_flutter
//
//  Copyright © 2023 DevCrew I/O
//

abstract class InvokeMethods {}

class OutputMethods implements InvokeMethods {
  static const openSnapCameraKit = "openSnapCameraKit";
  static const openSingleLens = "openSingleLens";
  static const getGroupLenses = "getGroupLenses";
}

class InputMethods implements InvokeMethods {
  static const cameraKitResults = "cameraKitResults";
  static const receivedLenses = "receivedLenses";
}
